osquery/tests/integration/tables/file.cpp

```diff
@@ -12,46 +12,70 @@
 // Sanity check integration test for file
 // Spec file: specs/utility/file.table
 
+#include <fstream>
+
 #include <osquery/tests/integration/tables/helper.h>
 
+#include <boost/filesystem.hpp>
+
 namespace osquery {
 
-class file : public IntegrationTableTest {};
-
-TEST_F(file, test_sanity) {
-  // 1. Query data
-  // QueryData data = execute_query("select * from file");
-  // 2. Check size before validation
-  // ASSERT_GE(data.size(), 0ul);
-  // ASSERT_EQ(data.size(), 1ul);
-  // ASSERT_EQ(data.size(), 0ul);
-  // 3. Build validation map
-  // See IntegrationTableTest.cpp for avaialbe flags
-  // Or use custom DataCheck object
-  // ValidatatioMap row_map = {
-  //      {"path", NormalType}
-  //      {"directory", NormalType}
-  //      {"filename", NormalType}
-  //      {"inode", IntType}
-  //      {"uid", IntType}
-  //      {"gid", IntType}
-  //      {"mode", NormalType}
-  //      {"device", IntType}
-  //      {"size", IntType}
-  //      {"block_size", IntType}
-  //      {"atime", IntType}
-  //      {"mtime", IntType}
-  //      {"ctime", IntType}
-  //      {"btime", IntType}
-  //      {"hard_links", IntType}
-  //      {"symlink", IntType}
-  //      {"type", NormalType}
-  //      {"attributes", NormalType}
-  //      {"volume_serial", NormalType}
-  //      {"file_id", NormalType}
-  //}
-  // 4. Perform validation
-  // EXPECT_TRUE(validate_rows(data, row_map));
+class File : public IntegrationTableTest {
```

Comment:
The name `File` is pretty common, it could be better to rename it to `FileTests` or to hide it in unnamed namespace.